crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -2759,9 +2759,15 @@ pub enum SuiInputObjectKind {
     SharedMoveObject {
         id: ObjectID,
         initial_shared_version: SequenceNumber,
+        #[serde(default = "bool_true")]
+        mutable: bool,
     },
 }
 
+const fn bool_true() -> bool {
```

Comment:
nit: can we name this something like `default_shared_object_mutability`?

Reply:
Renamed